app/src/main/java/com/kickstarter/viewmodels/ThreadViewModel.kt

```diff
@@ -68,15 +74,10 @@ interface ThreadViewModel {
                 .share()
 
             commentEnvelope
+                .compose(Transformers.combineLatestPair(comment))
                 .compose(bindToLifecycle())
                 .subscribe {
-                    Timber.i(
-                        "******* Comment envelope with \n" +
-                            "totalComments:${it.totalCount} \n" +
-                            "commentsList: ${it.comments?.map { comment -> comment.body() + " |"}} \n" +
-                            "commentsListSize: ${it.comments?.size} \n" +
-                            "pageInfo: ${it.pageInfoEnvelope} ****"
-                    )
+                    this.onCommentReplies.onNext(it.first.comments?.toCommentCardList(null))
```

Comment:
On line 86 I think you have already the project observable, you can add it to the RXChain so you can sent it when calling `toCommentCardList` instead of `null`